docs/asset-store/docs/05-04-asset-upload-service.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Asset Store Upload Service sub-chart
+type: Configuration
+---
+
+To configure the Asset Store Upload Service sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **replicaCount** | Defines the number of replicas of the service. | `1` |
+| **virtualservice.enabled** |  Enables to use an external service. | `false` |
```

Comment:
please specify that once you enable virtualservice it is available under https://{VIRTUALSERVICE_NAME}.{DOMAIN}/minio/, for example https://asset-upload-service.kyma.local/

Reply:
Changed to:
If you enable the **virtualservice**, it is available at `https://{VIRTUALSERVICE_NAME}.{DOMAIN}/`, for example `https://asset-upload-service.kyma.local/`.